src/EditorFeatures/Core.Cocoa/Microsoft.CodeAnalysis.EditorFeatures.Cocoa.csproj

```diff
@@ -1,7 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>net6.0-macos</TargetFramework>
     <DefineConstants>$(DefineConstants);COCOA</DefineConstants>
+    <!-- https://github.com/dotnet/roslyn/pull/59429#issuecomment-1043195174 -->
```

Comment:
Does this need a tracking bug?

Reply:
Thanks for the reminder, filed https://github.com/xamarin/xamarin-macios/issues/14331.